sdk-python/examples/interruption/README.md

```diff
@@ -0,0 +1,32 @@
+## Running a Interrupt Handler
+
+This example shows how to interrupt a workflow execution.
+You need to register and interrupt handler with thread.registerInterruptHandler.
+
+Let's run the example:
+
+```
+poetry shell
+python -m example_interruption
+```
+
+In another terminal, use `lhctl` to run the workflow:
+
+```
+# Run a new workflow
+lhctl run example-interrupt-handler
+
+# Take the resulting workflow ID, and do:
+lhctl get wfRun <wf run id>
+
+# Note that it is 'RUNNING'. Next, post an external event using the following:
+lhctl postEvent <wf run id> interruption-event NULL
+
+# Then inspect the wfRun:
+# Note it is 'COMPLETED' whit type 'INTERRUPT'
+lhctl get wfRun <wf run id>
+
+# Then inspect the output of the ExternalEvent and how it completed the nodeRun:
```

Comment:
This sounds incorrect. In an Interrupt scenario, `ExternalEvent` doesn't complete a `NodeRun`. Rather, the `ExternalEvent` causes the creation of a new `ThreadRun` with type `ThreadType.INTERRUPT`.

Also, on line 27, it may be `COMPLETD` or `RUNNING` depending on how fast you do `lhctl get wfRun`.

In order for the Interrupt to happen, the parent thread needs to be sleeping.